### ISSUE TITLE
NEW INDEX: TCBRI

### ISSUE THREAD

**Issue by remi-braun** [awesome-spectral-indices/awesome-spectral-indices#13]
```python
TCBRI=SpectralIndex(
    short_name='TCBRI',
    long_name='Tasseled Cap Brightness',
    formula='0.3037 *B + 0.2793 * G + 0.4743 * R + 0.5585 * N + 0.5082 * S1 + 0.1863 * S2',
    reference='10.1109/tgrs.1984.350619',
    type='vegetation',
    date_of_addition='2022-10-26',
    contributor="https://github.com/remi-braun"
)
```
See also the wikipedia page: https://en.wikipedia.org/wiki/Tasseled_cap_transformation
And the index database page: https://www.indexdatabase.de/db/r-single.php?id=723

**Comment by davemlz** [comment]
Hi @remi-braun,

About TC transformations: I need to think of how to standardize them since 1) they change according to the instrument, and 2) they change according to the processing level.

For now I will leave the issue open until I figure it :)

**Comment by remi-braun** [comment]
Yes no problem! You are managing these spectral indices in a more professional way than me 😉

**Comment by robmarkcole** [comment]
@davemlz any further thoughts on this? I've a need for use with Sentinel 2 soon :-)

**Comment by davemlz** [comment]
@robmarkcole actually yes! There is a new standard (breaking changes) being developed for ASI ;) but it will take a while (somewhen next year). I will keep you updated.

**Comment by davemlz** [comment]
Hey @remi-braun, @robmarkcole,

Took more than a year (sorry for that and thank you for the patience), but I'm finally migrating ASI to v1!

As part of that migration, I'm adding all TC proposed indices to the catalogue under the new v1 schema (experimental), where this type of indices are added under the `classification.family=tasseled_cap` property.

For more information, please check the migration guide here: https://awesome-spectral-indices.github.io/awesome-spectral-indices/v1.html

v1 schema is still experimental and it's subject to change (a new band standard will also come soon this month), but many properties are now quite solid! I'm expecting to release v1 around the end of fall this year. The APIs still work on the v0.x.x catalogue versions (and I will keep it like that for a while), so there is time for migrating software depending on them (@remi-braun, this may be important for https://github.com/sertit/eoreader), and in the meantime, I will add an experimental module pointing to v1.

Thanks a lot @remi-braun for teh contribution!

Best,

David
